platform/understand/externally-vs-platform-deployed.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Virtual Cluster Lifecycle
+sidebar_label: Virtual Cluster Lifecycle
+---
+
+# Virtual Cluster Lifecycle
+
+There are two main types of virtual clusters based on how they're deployed: platform-deployed and externally deployed. These classifications determine how the clusters are created, updated, and integrated with the platform's features. Understanding the differences between these types is important for effective cluster management and utilization of platform capabilities.
+
+## Platform-Deployed Virtual Clusters
+
+These are virtual clusters that are deployed and fully managed through our
+platform, regardless if they are deployed via the UI or CLI. They benefit from the platform's complete set of management features and automated lifecycle management.
+
+:::info Characteristics
+- Deployment methods:
+    - Platform UI
+    - vCluster CLI (when authenticated to Platform)
```

Comment:
Let's update this to 

Deployed using `vcluster platform create vcluster <name>` command